code/modules/mod/modules/modules_general.dm

```diff
@@ -655,3 +655,108 @@
 		has_tk = FALSE
 		return
 	mod.wearer.dna.remove_mutation(TK)
+
+//Special module for the Captain's Magnate suit, and the CC Corporate suit.
+//Literally only a thing so that https://pipe.miroware.io/5b52ba1d94357d5d623f74aa/mspfa/Nuke%20Ops/Panels/0648.gif can be real
+/obj/item/mod/module/hat_stabilizer
+	name = "MOD hat stabilizer module"
+	desc = "A simple set of deployable stands, directly atop one's head; \
+		these will deploy under a select few hats to keep them from falling off, allowing them to be worn atop the sealed helmet. Unfortunately, you still need to take off the hat first to deploy the helmet itself. \
+		This is a must-have for Nanotrasen Captains, enabling them to show off their authoritative hat even while in their MODsuit."
+	icon_state = "hat_holder"
+	module_type = MODULE_PASSIVE //The description says it auto-deploys for certain hats; that's just an excuse to have a whitelist of items.
+	complexity = 0 //Its FREE!
+	active_power_cost = 0 //Yes, thats right, FREE!
+	removable = FALSE
+	///Currently "stored" hat. No armor or function will be inherited, ONLY the icon.
+	var/obj/item/clothing/head/attached_hat
+	var/attachable_hats_list = list(/obj/item/clothing/head/caphat,)
+
+/obj/item/mod/module/hat_stabilizer/on_equip()
+	RegisterSignal(mod.helmet, COMSIG_PARENT_EXAMINE, .proc/add_examine)
+	RegisterSignal(mod.helmet, COMSIG_PARENT_ATTACKBY, .proc/place_hat)
+	RegisterSignal(mod.helmet, COMSIG_ATOM_ATTACK_HAND_SECONDARY, .proc/remove_hat)
+	attachable_hats_list = typecacheof(
```

Comment:
I mentioned it in the first comment but will mention it here:
Plasmamen have a PLASMAMAN_EXEMPT item flag (or something like that), rather than a list; this allows items to be blacklisted from placement.
The MOD helmet is thicker and generally bulkier than the plasmaman helmet, which means that most of the normally allowed items wouldn't work. As such, I'm using a whitelist, and testing each hat I put in it; that's why I specifically removed the Beret, and why the Traffic Cone aren't in - they dont fit properly, even if I mess with offsets.

I'll put it on init though, I was thinking of doing that anyways but just wanted to get the PR open for review